systematics_bindings.cpp

```diff
@@ -228,48 +391,161 @@ PYBIND11_MODULE(systematics, m) {
+        )mydelimiter")
+        .def("get_distance_to_root", [](sys_t & self, taxon_t * tax){return self.GetDistanceToRoot(tax);}, R"mydelimiter(
+            Given a taxon, this function calculates and returns the distance to its Most-Recent Common Ancestor (or its subtree root if none is found).
+            This function considers both unifurcations and extinct taxa. Consider using `get_branches_to_root()` instead for phylogeny reconstruction purposes.
+
+            Parameters
+            ----------
+            Taxon tax: Taxon to find distance to MRCA (or subroot) of.
+        )mydelimiter")
+        .def("get_variance_pairwise_distance", static_cast<double (sys_t::*) (bool) const>(&sys_t::GetVariancePairwiseDistance), R"mydelimiter(
+            This method calculates the variance of distance between all pairs of extant taxa. This is a measure of phylogenetic regularity (Tucker et. al., 2017).
+            This assumes the phylogenetic tree is fully connected. If this is not the case, it will return -1.
+
+            If `branch_only` is set, this method will only consider distances in terms of nodes that represent branches between two extant taxa. This is potentially useful as a comparison to real-world, biological data, where non-branching nodes cannot be inferred.
+
+            Parameters
+            ----------
+            bool branch_only: Only counts distance in terms of nodes that represent a branch between two extant taxa.
+        )mydelimiter")
+        .def("get_mean_pairwise_distance", static_cast<double (sys_t::*) (bool) const>(&sys_t::GetMeanPairwiseDistance), R"mydelimiter(
+        )mydelimiter")
+        .def("get_sum_pairwise_distance", static_cast<double (sys_t::*) (bool) const>(&sys_t::GetSumPairwiseDistance), R"mydelimiter(
+            This method calculates the mean distance between all pairs of extant taxa, also known as the Average Taxonomic Diversity. This is a measure of community distinctness (Webb and Losos, 2000; Warwick and Clark, 1998).
+            This assumes the phylogenetic tree is fully connected. If this is not the case, it will return -1.
+
+            If `branch_only` is set, this method will only consider distances in terms of nodes that represent branches between two extant taxa. This is potentially useful as a comparison to real-world, biological data, where non-branching nodes cannot be inferred.
+
+            Parameters
+            ----------
+            bool branch_only: Only counts distance in terms of nodes that represent a branch between two extant taxa.
+        )mydelimiter")
+        .def("get_phylogenetic_diversity", static_cast<int (sys_t::*) () const>(&sys_t::GetPhylogeneticDiversity), R"mydelimiter(
+            This method calculates the sum of edges of the Minimum Spanning Tree of the currently-active phylogenetic tree. Assuming all parent-child edges have a length of 1 (i.e., there are no unifurcations), this is a measure of phylogenetic diversity.
```

Comment:
Drop a citation here to Faith 1991 (?) (it's in the Empirical documentation)

Reply:
Done in db72793